website/docs/r/catalog_item.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_catalog_item"
+sidebar_current: "docs-vcd-resource-catalog-item"
+description: |-
+Provides a vCloud Director Catalog item resource. This can be used to upload and delete OVA file in catalog.
+---
+
+# vcd\_catalog\_item
+
+Provides a vCloud Director Catalog item resource. This can be used to upload OVA to catalog and delete it.
+
+Supported in provider *v2.0+*
+
+## Example Usage
+
+```
+resource "vcd_catalog_item" "myNewCatalogItem" {
+  org = "my-org"
+  catalog = "my-catalog" 
+
+  name = "my ova"
+  description = "new vapp template"
+  ova_path = "/home/user/file.ova"
+  upload_piece_size = 10 
+  show_upload_progress = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, isn't needed if defined/match at provider level
```

Comment:
"The name of organization to use, isn't needed if defined/match at provider level" ==> 
"The name of organization to use. Not needed if defined at provider level"